### PR TITLE
Sofia/cleanup condition expr

### PR DESCRIPTION
**Pull request by sofiiako**

### CODE REVIEW

File: Condition expressions/If statement/task.md [hunk 1]
```diff
@@ -1,7 +1,20 @@
 ## If statement
 
-The `if` keyword is used to form a conditional statement that executes some specified code after checking if its expression is `True` . Python uses indentation to define code blocks.  
-  
+Compound statements in Python contain (groups of) other statements; they affect or control 
+the execution of those other statements in some way. The `if`, `while` and `for` statements
```
Comment (Geravant) [inline Condition expressions/If statement/task.md:4]: We are nott actually talking in tterms of control flow here in the lesson, so we probably need o loose the mention of the loops and simplify the description

File: Condition expressions/If statement/task.md [hunk 2]
```diff
@@ -1,7 +1,20 @@
...
+implement traditional control flow constructs.
+
+Perhaps the most well-known statement type is the if statement. The `if` keyword is 
+used to form a conditional statement that executes some 
+specified code after checking if its expression is `True`. Python uses indentation
```
Comment (Geravant) [inline Condition expressions/If statement/task.md:9]: We can elaborate on the exact  indentation  type used here

File: Condition expressions/Else and elif statements/task.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+## Else and elif statements
+
+The `elif` and `else` statements complement the `if` statement. 
+There can be zero or more `elif` parts, and the `else` part is optional. The keyword
```
Comment (Geravant) [inline Condition expressions/Else and elif statements/task.md:4]: We can mention them being substitute for a switch somewhere

File: Condition expressions/lesson-info.yaml [hunk 4]
```diff
@@ -2,4 +2,4 @@ content:
 - Boolean operators
 - Boolean operators order
 - If statement
-- Else, elif part in if statement
+- Else and elif statements
```
Comment (Geravant) [inline Condition expressions/lesson-info.yaml:5]: Why are we losing the word part?

File: Data structures/Tuples/task-info.yaml [hunk 5]
```diff
@@ -6,5 +6,8 @@ files:
   - offset: 161
     length: 13
     placeholder_text: alphabet length
+  - offset: 176
+    length: 0
```
Comment (Geravant) [inline Data structures/Tuples/task-info.yaml:10]: Do we need a placeholder withou a length? What are we trying to ask?

File: Introduction/Comments/task-info.yaml [hunk 6]
```diff
@@ -3,8 +3,8 @@ files:
 - name: comments.py
   visible: true
   placeholders:
-  - offset: 138
-    length: 11
+  - offset: 137
+    length: 12
     placeholder_text: add new comment here
```
Comment (Geravant) [inline Introduction/Comments/task-info.yaml:8]: May be for a demonstraion purpose we can omit the number sign here

**Review by Geravant — CHANGES_REQUESTED** [review]

File: Condition expressions/If statement/if_statement.py [hunk 7]
```diff
@@ -1,11 +1,11 @@
 name = "John"
 age = 17
 
-if name == "John" or age == 17:   # check that name is "John" or age is 17. If so print next 2 lines.
+if name == "John" or age == 17:  # check that name is "John" or age is 17. If so print next 2 lines.
```
[the remark this answers is not in the document]
Comment (sofiiako) [reply Condition expressions/If statement/if_statement.py:4]: It comes from the original version) But agreed

**Review by Geravant — APPROVED** [review]

**Pull request merged**